clang/test/Analysis/nullability.c

```diff
@@ -1,4 +1,6 @@
-// RUN: %clang_analyze_cc1 -fblocks -analyzer-checker=core,nullability -Wno-deprecated-non-prototype -verify %s
+// RUN: %clang_analyze_cc1 -fblocks -analyzer-checker=core,nullability,debug.ExprInspection -Wno-deprecated-non-prototype -verify %s
```

Comment:
Consider eliminating the use of  `-Wno-deprecated-non-prototype` by specifiying the parameter lists of functions. (Note that this is C, so you need `return_t funcname(void)` to specify a function that takes no arguments.)

Reply:
Done in a9c489f683fc